### PR TITLE
Remove extraneous elses

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Further removal of elses directly after a return

Same as https://github.com/civicrm/civicrm-core/pull/19168 but a different file

This looks like a huge code change but it's just removing meaningless elses and reformatting view using 
https://github.com/civicrm/civicrm-core/pull/19171/files?w=1 for better readability

Before
----------------------------------------
Constructs like

```
if ($a = 1) {
  return;
}
else {
  // do the thing
}
```

After
----------------------------------------

```
if ($a = 1) {
  return;
}
 // do the thing

```

Technical Details
----------------------------------------

### CODE REVIEW

File: CRM/Contribute/Import/Parser/Contribution.php [hunk 1]
```diff
@@ -335,21 +335,19 @@ public function import($onDuplicate, &$values) {
...
+        foreach ($dupeIds as $k => $v) {
+          if ($v) {
+            $errorMsg[] = "$labels[$k] $v";
+          }
         }
+        $errorMsg = implode(' AND ', $errorMsg);
+        array_unshift($values, 'Matching Contribution record not found for ' . $errorMsg . '. Row was skipped.');
+        return CRM_Import_Parser::ERROR;
       }
     }
```
Comment (eileenmcnaughton) [inline CRM/Contribute/Import/Parser/Contribution.php:354]: This looks wonky in web view but

File: CRM/Contribute/Import/Parser/Contribution.php [hunk 2]
```diff
@@ -367,108 +365,100 @@ public function import($onDuplicate, &$values) {
...
+            return CRM_Import_Parser::ERROR;
           }
+        }
 
-          // process pledge payment assoc w/ the contribution
-          return self::processPledgePayments($formatted);
+        $this->_newContributions[] = $newContribution['id'];
+        $formatted['contribution_id'] = $newContribution['id'];
 
-          return CRM_Import_Parser::VALID;
```
Comment (eileenmcnaughton) [inline CRM/Contribute/Import/Parser/Contribution.php:399]: double return

**Comment by eileenmcnaughton** [comment]
sigh @seamuslee001 all these files are hard to make 'the change' in

**Comment by seamuslee001** [comment]
If Jenkins likes it then I do too

**Pull request merged**